alertingest/storage.py

```diff
@@ -80,8 +80,8 @@ def __init__(
         self,
         alert_bucket_name: str,
         schema_bucket_name: str,
-        s3_client: Any | None = None,
-        endpoint_url: str | None = None,
+        s3_client: Union[Any, None] = None,
```

Comment:
In python 3.10+, the correct way to specify union types is with the `|` operator as you had it previously. Why have you switched to the older `Union[]` types?